flow/probes/gopacket.go

```diff
@@ -175,12 +177,14 @@ func (p *GoPacketProbe) Run(packetCallback func(gopacket.Packet), e FlowProbeEve
 		}
 	}
 
+	metadata := &CaptureMetadata{}
+	e.OnStarted(metadata)
```

Comment:
this e != nil is just after, maybe useless now ?

Reply:
Yes, probably. Removed